notify_center.go

```diff
@@ -176,7 +181,7 @@ func (nc *NotifyCenter) startConsumerGroupEvaluator(group string, cluster string
 		nc.groupLock.RUnlock()
 
 		// Send requests for group status - responses are handled by the main loop (for now)
-		storageRequest := &RequestConsumerStatus{Result: nc.resultsChannel, Cluster: cluster, Group: group}
+		storageRequest := &RequestConsumerStatus{Result: nc.resultsChannel, Cluster: cluster, Group: group, Showall: true}
```

Comment:
Whoa, this is a huge change and affects all notifiers. Every other notifier does is not interested in getting the entire dump of partitions, just the ones in error. If you want to have a notifier perform a full dump like this, it needs to be specified per-notifier.

I suggest extending the notifier interface to provide this boolean, and having it default to false unless the notifier overrides it.